features/step_definitions/networking.rb

```diff
@@ -881,9 +881,24 @@
 Given /^the Internal IP of node "([^"]*)" is stored in the#{OPT_SYM} clipboard$/ do |node_name,cb_ipaddr|
   ensure_admin_tagged
   node = node(node_name)
+  _admin = admin
```

Comment:
what is the point to create this local variable?

Reply:
Other than that lgtm.

Reply:
Indeed we don't need it, Thanks for catching that redundancy. Fixed.